portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -40,21 +40,29 @@
   private static final String comment = "Comment";
   private static final String content = "content";
   private static final String textInput = "text-input";
+  private static final String quantity = "quantity";
 
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String maxString = request.getParameter("maxComment");
+    int max = Integer.parseInt(maxString);
```

Comment:
I changed it so that if no MaxComment is in the param, the string is auto set to 2 which is the original default value. also, since no maxcomment value outside of 1-5 is allowed to even be submitted, nothing else should be passed.

Reply:
Is there any validation preventing the submission that isn't in the javascript or HTML?

Those validations are trivially overrideable, e.g. by editing the JS or HTML, or just by doing a direct request to the servlet /data)./

In general, never trust any data or validation that comes from a browser. :)

Reply:
Okay that makes sense! I just added an if statement in DataServlet.java that controls for this and sets a default value.